docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -227,27 +227,17 @@ properties only apply to a ``TASK`` retry policy.
    * - Property name
      - Description
      - Default value
-   * - ``fault-tolerant-execution-target-task-input-size``
```

Comment:
What's the behavior for users who still have this (and other removed properties) configured?

Reply:
server startup will fail. Let me add this property to `DefunctConfigs`